t3_lazy_couch_potato_v0: #namespace
  agent_name: qlearn #dqn
  agent_class: QLearn #DeepQNetwork 